docs/tutorial/work-pools.md

```diff
@@ -178,166 +140,145 @@ abc/workspace/123/deployments/deployment/xyz
+    from prefect import flow                                                       
+    from prefect.deployments import DeploymentImage                                
 
-To take advantage of this functionality, you can write your deploy script like this:
 
-```python hl_lines="14" title="example_deploy_script.py"
-from prefect import flow                                                       
-from prefect.deployments import DeploymentImage                                
+    @flow(log_prints=True)
+    def my_flow(name: str = "world"):
+        print(f"Hello {name}! I'm a flow running on Cloud Run!")
 
 
-@flow(log_prints=True)
-def my_flow(name: str = "world"):
-    print(f"Hello {name}! I'm a flow running on Cloud Run!")
-
-
-if __name__ == "__main__":                                                     
-    my_flow.deploy(                                                            
-        name="my-deployment",
-        work_pool_name="above-ground",
-        cron="0 1 * * *",
-        image=DeploymentImage(
-            name="my-image:latest",
-            platform="linux/amd64",
+    if __name__ == "__main__":                                                     
+        my_flow.deploy(                                                            
+            name="my-deployment",
+            work_pool_name="above-ground",
+            cron="0 1 * * *",
+            image=DeploymentImage(
+                name="my-image:latest",
+                platform="linux/amd64",
+            )
         )
-    )
-```
-
-Run the script to create the deployment on the Prefect Cloud server.
-
-Running this script will build a Docker image with the tag `<region>-docker.pkg.dev/<project>/<repository-name>/my-image:latest` and push it to your repository.
-
-!!! tip
-    Make sure you have Docker running locally before running this script.
+    ```
 
-Note that you only need to include an object of the `DeploymentImage` class with the argument `platform="linux/amd64` if you're building your image on a machine with an ARM-based processor.
-Otherwise, you could just pass `image="my-image:latest"` to `deploy`.
+    This script accomplishes the following:
 
-Also note that the `cron` argument will schedule the deployment to run at 1am every day. 
-See the [schedules](/concepts/schedules/) docs for more information on scheduling options.
+    * Creates a deployment on the Prefect Cloud server.
+    * Builds a Docker image with the tag `<region>-docker.pkg.dev/<project>/<repository-name>/my-image:latest` and pushes it to your repository.
+    * If you're building your image on a machine with an ARM-based processor, you only need to include an object of the `DeploymentImage` class with the `platform="linux/amd64` argument. Otherwise, you could just pass `image="my-image:latest"` to `deploy`.
+    * The `cron` argument schedules the deployment to run at 1 AM every day. For information on scheduling options, see [schedules](/concepts/schedules/).
```

Comment:
These last two bullets don't feel like they belong in this list. They are relevant details for sure, but not things that the script accomplished.